lib/virt_autotest/utils.pm

```diff
@@ -1132,10 +1143,10 @@ sub subscribe_extensions_and_modules {
 
   is_sev_es_guest($guest_name)
 
-Check whether a guest is sev-es, sev only or not sev/sev-es guest by searching
-whether sev-es or sev word is available in its name. The only argument for the
-subroutine is guest_name. It returns sev-es or sev if either is found in guest
-name, otherwise it returns 0.
+Check whether a guest is sev-snp, sev-es, sev only, or not sev/sev-es/sev-snp guest by searching
+whether sev-snp, sev-es, or sev word is available in its name. The only argument for the
+subroutine is guest_name. It returns sev-snp, sev-es, or sev if any is found in guest
+name, otherwise it returns 'notsev'.
```

Comment:
If it is extended to tell sev-snp , will it still suit to have the function name `is_sev_es_guest`?

Reply:
I removed such change, I didn't use this function in the code.